include/wx/event.h

```diff
@@ -2944,6 +2950,7 @@ class WXDLLIMPEXP_CORE wxUpdateUIEvent : public wxCommandEvent
     virtual wxEvent *Clone() const wxOVERRIDE { return new wxUpdateUIEvent(*this); }
 
 protected:
+    bool          m_isCheckable;
```

Comment:
For consistency with the initialization order I'd put this after `m_setChecked`.